### PR TITLE
Added Google Auth 

### PR DESCRIPTION
**Pull request by Abh1shekSingh**
# Description

Added Google Authentication 

Fixes:  #809

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] I have made this from my own
- [x] I have taken help from some online resourses 
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK

### CODE REVIEW

**Comment by Abh1shekSingh** [comment]
@VidhiBhatt01 Resolved the conflicts. Please review and merge

**Review by VidhiBhatt01 — COMMENTED** [review]
Why so?
image

**Comment by VidhiBhatt01** [comment]
https://deploy-preview-832--cool-front-end-templates.netlify.app/login%20form/

**Comment by VidhiBhatt01** [comment]
Also, agar style_link.json mein add na kara ho to woh bhi kar do

**Comment by Abh1shekSingh** [comment]
@VidhiBhatt01  jab PR merge Ho jaygi, tab Unauthorised domain wala error nhi ayga. Coolfront ki Doman url mene provide krwayi hai firebase mein isiliye. Yeh random deploy link jo GitHub deta hai, just to check PR yeh nhi dala Na.

File: Login form/index.html [hunk 1]
```diff
@@ -261,7 +261,6 @@ <h3>One of us ?</h3>
     }else{
       document.getElementById('useravatar').src = "";
       document.getElementById('username').innerHTML = "";
-      document.getElementById('username').innerHTML = "";
```
Comment (arpit456jain) [inline Login form/index.html:264]: @Abh1shekSingh since there is no major change in this PR that's why i can't give lvl2

**Comment by Abh1shekSingh** [comment]
Bhai their is a Major change. But conflicts aa rhe the, isiliye mene sara content copy paste krne padhe the bhai , new PR bnai thi bhai, aur Changes Show krne ke liye

**Pull request merged**